spinnaker-monitoring-daemon/spinnaker-monitoring/newrelic_service.py

```diff
@@ -75,7 +75,8 @@ def extract_tags(options):
         option_tags = options['newrelic'].get('tags', []) or []
         for tag in option_tags:
             if tag.find(":"):
-                tags.update(tag.split(":", 2))
+                key, value = tag.split(":", 2)
```

Comment:
While you're updating this, should this actually be `tag.split(":", 1)`, which would return an array of max size 2 rather than `tag.split(":", 1)`, which would return an array of max size 3?

Reply:
@ezimanyi updated to return a max size of `1`. re: your first comment, my initial change updated line 78 to `key, value = tag.split(":", 2)` removing the `tags.update` call. It should now read `key, value = tag.split(":", 1)`. Let me know if you do not see the same (or if I'm looking in the wrong place).

Reply:
Sorry, I think I got confused and thought the same pattern was twice in this file---you did indeed update the line that I mentioned above.  Looks good, thanks!

Reply:
Thank *you*!